src/orchestrator/core/mongo.py

```diff
@@ -58,6 +59,20 @@ def createIndexes(self, SERVICE_NAME):
         except Exception, e:
             logger.warn("createIndex database %s exception: %s" % (databaseName,e))
 
+
+    def createSTHIndexes(self, SERVICE_NAME, SUBSERVICE_NAME):
+        try:
+            # For STH
```

Comment:
I understand the index for raw collection should be also created, in a similar way. In this case the index structure is:

```
{entityId: 1, entityType: 1, attrName: 1, recvTime: -1}
```

In addition, I'd suggest to include a link in this part to https://github.com/telefonicaid/fiware-sth-comet/blob/master/doc/manuals/db_indexes.md, for reference to know where these indexes come.

Reply:
fixed in https://github.com/telefonicaid/orchestrator/pull/204